lib/snowflake/index.js

```diff
@@ -3,6 +3,34 @@ const snowflake = require('snowflake-sdk')
 const Log = require('@kelchy/log')
 
 class Snowflake extends EventEmitter {
+  /**
+   *
+   * @param {string} connStr - format: "snowflake://<username>:<password>@<account>.snowflakecomputing.com/?role=<role>&warehouse=<warehouse>&database=<database>&schema=<schema>"
+   * @returns snowflake credentials obj
+   */
+  static parseConnStr (connStr) {
```

Comment:
can you move this function below the constructor? as best practice, constructors are always the first definition on a class